static/openapi/vwrs-spec.json

```diff
@@ -467,6 +467,70 @@
             "example": "test-service-us-east.example.io"
           }
         }
+      },
+      "GetStatusResponse": {
+        "type": "object",
+        "properties": {
+          "avg_waiting_time": {
+            "type": "number",
+            "format": "float",
+            "description": "The average time users spend in the virtual waiting room before being granted access to the origin service (in milliseconds).",
+            "example": 19871.8
+          },
+          "backoff_interval": {
+            "type": "integer",
+            "description": "Indicates the interval in seconds after which the EdgeWorker should call the status API call to the vwrs server. This is used to reduce server load.",
+            "example": 1
+          },
+          "position": {
+            "type": "integer",
+            "description": "The client's current position in the waiting room.",
+            "example": 99359
+          },
+          "queue_depth": {
+            "type": "integer",
+            "description": "Total number of users present in the waiting room.",
+            "example": 99359
+          },
+          "rate_limit": {
+            "type": "integer",
+            "description": "The rate limit for the domain. The maximum users per second allowed to reach the origin.",
+            "example": 5
+          },
+          "waiting_room_interval": {
+            "type": "integer",
+            "description": "The time interval in seconds when the UI reloads itself to get the waiting room status.",
+            "example": 1
+          }
+        }
+      },
+      "GetAccessResponse": {
+        "type": "object",
+        "required": [
+          "position"
+        ],
+        "properties": {
+          "position": {
+            "type": "number",
+            "description": "Position in the waiting room. 0 means the user doesn't need to wait in waiting room any longer and can go to the origin."
+          },
+          "waiting_time": {
+            "type": "number",
+            "description": "Expected time to spend in waiting room (in seconds)."
+          },
+          "queue_depth": {
+            "type": "number",
+            "description": "Number of users currently in the waiting_room."
+          },
+          "visitor_id": {
+            "type": "number",
+            "description": "The user's session ID for this access request."
+          },
+          "refresh_interval": {
+            "type": "number",
+            "description": "Specifies the time interval, in seconds, for subsequent checks of the session status."
```

Comment:
And here's a third way we are stating the time unit.

Reply:
@todor-macrometa can we change visitior_id to request_id to be consistent with the status API below this one.